$NetBSD: patch-Makefile,v 1.1 2012/11/17 11:50:14 ryoon Exp $

- Install Coccilib man page just as Coccilib.3 (no need to have a custom
  `cocci' suffix)
- Do not install pyml bundle

--- Makefile.orig	2019-09-25 09:19:54.000000000 +0000
+++ Makefile
@@ -297,10 +297,6 @@ install-bash:
 install-spatch : spatch$(TOOLS_SUFFIX)
 	$(MKDIR_P) $(DESTDIR)$(BINDIR) $(DESTDIR)$(LIBDIR)
 	$(MKDIR_P) $(DESTDIR)$(LIBDIR)/ocaml
-	if test -f bundles/pyml/dllpyml_stubs.so; then \
-		$(INSTALL_PROGRAM) bundles/pyml/dllpyml_stubs.so \
-			$(DESTDIR)$(LIBDIR); \
-	fi
 	if test -f bundles/pcre/dllpcre_stubs.so; then \
 		$(INSTALL_PROGRAM) bundles/pcre/dllpcre_stubs.so \
 			$(DESTDIR)$(LIBDIR); \
@@ -318,7 +314,7 @@ install-man :
 	$(INSTALL_DATA) docs/spatch.1 $(DESTDIR)$(MANDIR)/man1/
 	$(INSTALL_DATA) docs/pycocci.1 $(DESTDIR)$(MANDIR)/man1/
 	$(INSTALL_DATA) docs/spgen.1 $(DESTDIR)$(MANDIR)/man1/
-	$(INSTALL_DATA) docs/Coccilib.3cocci $(DESTDIR)$(MANDIR)/man3/
+	$(INSTALL_DATA) docs/Coccilib.3cocci $(DESTDIR)$(MANDIR)/man3/Coccilib.3
 
 .PHONY : install-spgen
 install-spgen : tools/spgen/source/spgen$(TOOLS_SUFFIX)
